modules/katamari/src/main/ts/ephox/katamari/api/Type.ts

```diff
@@ -20,12 +20,34 @@ const isSimpleType = <Yolo>(type: string) => (value: any): value is Yolo =>
 const eq = <T> (t: T) => (a: any): a is T =>
   t === a;
 
+const getPrototypeOf = Object.getPrototypeOf;
```

Comment:
This should be at the top of the file, not in the middle like this.

Reply:
Done